tsl/test/expected/continuous_aggs.out

```diff
@@ -1738,3 +1738,107 @@ SELECT
 --
 (0 rows)
 
+-- test that option create_group_indexes is taken into account
+SET client_min_messages = ERROR;
+CREATE TABLE test_group_idx (
+time timestamptz,
+symbol int,
+value numeric
+);
+select create_hypertable('test_group_idx', 'time');
+      create_hypertable       
+------------------------------
+ (48,public,test_group_idx,t)
+(1 row)
+
+insert into test_group_idx
+(select generate_series(
+	'2020-01-01',
+    '2020-02-25', INTERVAL '30 sec'
+  ),
+  round(random()*10),
+  random()*5);
+create materialized view cagg_index_true
+with (timescaledb.continuous, timescaledb.create_group_indexes = true) as 
+select
+	time_bucket('1 day', "time") as bucket,
+	sum(value),
+	symbol 
+from test_group_idx 
+group by bucket, symbol;
+create materialized view cagg_index_false
+with (timescaledb.continuous, timescaledb.create_group_indexes = false) as 
+select
+	time_bucket('1 day', "time") as bucket,
+	sum(value),
+	symbol 
+from test_group_idx 
+group by bucket, symbol;
+create materialized view cagg_index_default
+with (timescaledb.continuous) as 
+select
+	time_bucket('1 day', "time") as bucket,
+	sum(value),
+	symbol 
+from test_group_idx 
+group by bucket, symbol;
+-- make sure a group index has been created when explicitly asked for
+SELECT format('%I.%I', materialization_hypertable_schema,
+materialization_hypertable_name) AS mat_hyper_cagg_index_true
+FROM timescaledb_information.continuous_aggregates where view_name like 'cagg_index_true'
+\gset
+\d+ :mat_hyper_cagg_index_true
+                          Table "_timescaledb_internal._materialized_hypertable_49"
+  Column  |           Type           | Collation | Nullable | Default | Storage  | Stats target | Description 
+----------+--------------------------+-----------+----------+---------+----------+--------------+-------------
+ bucket   | timestamp with time zone |           | not null |         | plain    |              | 
+ agg_2_2  | bytea                    |           |          |         | extended |              | 
+ symbol   | integer                  |           |          |         | plain    |              | 
+ chunk_id | integer                  |           |          |         | plain    |              | 
+Indexes:
+    "_materialized_hypertable_49_bucket_idx" btree (bucket DESC)
+    "_materialized_hypertable_49_symbol_bucket_idx" btree (symbol, bucket DESC)
+Triggers:
+    ts_insert_blocker BEFORE INSERT ON _timescaledb_internal._materialized_hypertable_49 FOR EACH ROW EXECUTE FUNCTION _timescaledb_internal.insert_blocker()
+Child tables: _timescaledb_internal._hyper_49_106_chunk,
+              _timescaledb_internal._hyper_49_107_chunk
```

Comment:
I'm always wondering about it and why we don't define a standard because there are several places using test support functions like `test.show_columns` and `test.show_columnsp` (test/sql/utils/testsupport.sql).